docs/specs/git.yml

```diff
@@ -100,6 +100,34 @@ outputs:
   build.manifest:
   dep.json:
 ---
+### Author is case-insensitive
```

Comment:
is there an existing test we can modify? git tests are slower to run, I'm trying to reduce the total e2e test time.